vipps-ecom-api.md

```diff
@@ -235,6 +281,41 @@ that Vipps can call to delete the data. The Vipps app allows the user to later
 remove this consent (via the Profile -> Security -> "Access to your information"
 -> "Companies that remember you" screens).
 
+## Userinfo
+
+```
+Early draft, this should be considered pilot functionality that we are currently rolling out in our test environemnt.
+```
+
+Vipps offers a functionality to ask for a generic conset to access Userinfo. This is based on the [Vipps Login](https://github.com/vippsas/vipps-login-api) solution, but you as a merchant can seemlessly combine the two functionalites in a single user session. Combining both the userinfo and payment elements. This means that a merchant needs be registered with both Vipps Login, and Vipps Ecom functionality.
+
+When you initiate a payment add the parameter `Scopes` to ask for a users consent to share these details. For example like be email, address and name. The scopes are based on [Vipps Login's list of socopes](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#scopes).
```

Comment:
are we using a capital in `Scopes` or `scopes`

Reply:
Non capital, changing

Reply:
correct

Reply:
Maybe we also should use the singular form `scope` as the oidc spec (and userinfo after my PR) does? Or maybe it's for the best to use a more explicit naming like `userinfo_scope` or `consent_scope` as `scope` might be confusing in a transaction/payment context.?

https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims
https://tools.ietf.org/html/rfc6749#section-3.3